src/Application/Renderer/Template/Action/SourcecodeHandler.php

```diff
@@ -37,7 +37,7 @@ public function __construct(Analyzer $analyzer)
      */
     public function __invoke(Action $action)
     {
-        $artifact = $action->getRenderPass()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
+        $artifact = $action->getRenderContext()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
```

Comment:
While being in favour of the British spelling, I'm afraid that it will be slightly hard to change, as I see "artifact" being used in the xml templates as well.

But the final call is on you. I'd say go for it if you're up to it.